src/components/evaluation/EvaluationContent.tsx

```
@@ -0,0 +1,104 @@
+            ))}
+          </S.ScrollArea>
+        </S.MainContent>
+
+        <S.SidebarRight>
+          <S.CompletedTitle>평가완료</S.CompletedTitle>
+          {memberList
+            .filter((memberData) => memberData.evaluated)
+            .map((memberData) => (
+              <S.CompletedButton>{memberData.nickname}</S.CompletedButton>
+            ))}
+        </S.SidebarRight>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)